src/constants.js

```diff
@@ -0,0 +1,5 @@
+export const API_KEY = '8cac6dec66e09ab439c081b251304443'
```

Comment:
Security Issue: API_KEY is exposed in the code (line 1)
   - Store in a secure environment variable (e.g., .env file)